02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Barbara Jo Dukette</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+    <h1>Barbara Jo Dukette</h1>
+    <h4><em>507 Pressler, Austin, Texas</em></h4>
+
+    <Section><h2>Bio</h2>
+        <p>I was once a <strong>little girl</strong>. Then I was a young woman. I was once a middle-aged woman, and I woke up one morning
+          looking at 60. It's scary. But if you think of the alternative, that is, death, I guess it beats that. Why is this paragraph
+          not wrapping? Do I need to insert a paragraph command? Let's see. Yes, I needed to insert a paragraph start and finish.
+          I'm a bit confused as to what the point of a "section" is.</p>
+
+    <h2>Hobbies</h2>
+        <ol>
+            <li>Climbing Trees (when I was a little girl)</li>
+            <li>Dancing (when I was a young woman)</li>
+            <li>Cross-stitching (when I was a middle-aged woman)</li>
+            <li>Geocaching (in this time of my life)</li>
+        </ol>
+      <h2>Things I Don't Like</h2>
+        <ul>
+            <li>Repetitious work</li>
+            <li>Repetitious people</li>
+            <li>Repetitious ideas</li>
+        </ul>
+
+      <h2>TV Shows I like to watch</h2>
+        <img src="http://netflixlife.com/files/2015/01/Doctorwho_50th-anniversary-thumbnail_01.jpg" height="171" width"85" />
+      <p>
+        <img src="http://www.hercampus.com/sites/default/files/2016/05/02/unbreakable-kimmy-schmidt-poster.jpg"
+      /><p>
+      <h2>Favorite Websites</h2>
+            <ul><li><a href="https://www.geocaching.com" >Geocaching.com</a></li>
+            <li><a href="https://google.com" >Google.com</a></li>
+          </ul>
+
+  </section>
 
 </body>
 </html>
```

Comment:
Great work, Barbara! Geocaching looks fun. Have you gotten any cool stuff from it?